WordPressVIPMinimum/Tests/Functions/StripTagsNotAllowedUnitTest.inc

```diff
@@ -0,0 +1,3 @@
+<?php
+strip_tags( $foo, $bar ); // not-ok
+strip_tags( $foo ); // not-ok
```

Comment:
How about:

```php
strip_tags( 'Foo, Bar' );
strip_tags( 'Foo' . ', ' . 'Bar' );
strip_tags( 'Foo, Bar', $bar );
```
?

Reply:
Agreed, @justnorris can you make the change?

Reply:
I missed this. I'll add this to my todo list.